.gitlab-ci.yml

```
@@ -209,6 +209,14 @@ cwl_v1.2:
     - python setup_gitlab_docker.py  # login to increase the docker.io rate limit
     - make test tests=src/toil/test/cwl/cwlTest.py::CWLv12Test
 
+cwl_on_arm:
+  stage: main_tests
+  script:
+    - pwd
+    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[cwl,aws]
+    - python setup_gitlab_docker.py  # login to increase the docker.io rate limit
+    - make test tests=src/toil/test/cwl/cwlTest.py::CWLOnARMTest
```

Actually, the tests are failing because we don't actually go get the private keys we need in order to log into the AWS instances we make, [like we do for the integration tests](https://github.com/DataBiosphere/toil/blob/c758dd2986281b2c9a95c171ecd162263e1cb358/.gitlab-ci.yml#L295-L297). We need to run the magic key-getting script:

```suggestion
    - pwd
    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[cwl,aws]
    - python setup_gitlab_docker.py  # login to increase the docker.io rate limit
    # This reads GITLAB_SECRET_FILE_SSH_KEYS
    - python setup_gitlab_ssh.py
    - chmod 400 /root/.ssh/id_rsa
    - make test tests=src/toil/test/cwl/cwlTest.py::CWLOnARMTest
```